src/components/InfoTip/InfoTip.stories.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+import { DOCUMENTATION_URL } from '../../../storybook/constants';
+import { ListGroup, ListGroupItem } from '../ListGroup';
+import { Icon } from '../Icon';
+
+import { InfoTip } from './index';
+
+const stories = storiesOf('InfoTip', module);
+
+stories.addDecorator(
+  defaultTemplate({
+    title: 'InfoTip',
+    documentationLink: DOCUMENTATION_URL.PATTERNFLY_ORG_WIDGETS + '#info-tip'
+  })
+);
+
+stories.addWithInfo('InfoTip', '', () => (
+  <nav className="navbar navbar-default navbar-pf" role="navigation">
+    <div className="navbar-header">
+      <button
+        type="button"
+        className="navbar-toggle"
+        data-toggle="collapse"
+        data-target=".navbar-collapse-1"
+      >
+        <span className="sr-only">Toggle navigation</span>
+        <span className="icon-bar" />
+        <span className="icon-bar" />
+        <span className="icon-bar" />
+      </button>
+      <a className="navbar-brand" href="/">
+        <img
+          src="http://www.patternfly.org/assets/img/brand.svg"
+          alt="PatternFly Enterprise Application"
+        />
+      </a>
+    </div>
+    <div className="collapse navbar-collapse navbar-collapse-1">
+      <ul className="nav navbar-nav navbar-utility">
+        <InfoTip id="infotip-widget">
+          <InfoTip.InfoTipToggle bsRole="toggle">
+            Messages: 2
+          </InfoTip.InfoTipToggle>
+          <InfoTip.InfoTipMenu bsRole="menu">
```

Comment:
Shouldn't the bsRole="toggle" and bsRole="menu" supposed to be the default behavior?
I don't think consumers should specify them.

Reply:
If I remove bsRole or is et by default behaviour this fail, it seems that Dropdown needs to know where are the toggle and menu